inductiva/tasks/task.py

```diff
@@ -153,14 +154,14 @@ def set_output_class(self, output_class):
         """Set the output class of the task."""
         self._output_class = output_class
 
-    def get_output(self, output_dir=None):
+    def get_output(self, output_dir: Optional[types.Path] = None):
         """Get the output of the task.
 
         Returns:
             The output of the task.
         """
         self.wait()
```

Comment:
do you want to info the user that you are getting the input but you are now having to wait for the task to be finished?

Don't forget to provide as much feedback to the user as possible.

Reply:
I've updated the comments.